### PR TITLE
MNIST

### PR DESCRIPTION
**Pull request by madhavajay**
## Description
Started on required MNIST functionality as a way to understand adding completely new arbitrary classes and methods.
This was mostly a learning session for me as I went through the full code path of install_inside_library and the allowlist.

Added a debugging notebook that uses inspect to walk objects and print out their attributes recursively.

## Affected Dependencies
None

## How has this been tested?
Locally.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: src/syft/lib/torch/allowlist.py [hunk 1]
```diff
@@ -617,3 +617,10 @@
 # allowlist.add("torch.nn.Linear.parameters")
 allowlist["torch.nn.parameter.Parameter"] = "torch.nn.parameter.Parameter"
 allowlist["torch.nn.parameter.Parameter.__add__"] = "torch.nn.parameter.Parameter"
+
+
+# MNIST
+allowlist["torch.manual_seed"] = "torch.Generator"
+allowlist["torch.Generator"] = "torch.Generator"
+allowlist["torch.Generator.get_state"] = "torch.Tensor"  # Probably dont want this?
```
Comment (iamtrask) [inline src/syft/lib/torch/allowlist.py:625]: No reason to not include it :)

**Review by iamtrask — APPROVED** [review]

File: src/syft/lib/torch/allowlist.py (hunk 1 above)
Comment (madhavajay) [reply src/syft/lib/torch/allowlist.py:625]: Is there any thing insecure about getting or setting the state of a random number generator?
